sops/__init__.py

```diff
@@ -202,10 +207,17 @@ def main():
         else:
             print("%s doesn't exist, creating it." % args.file)
 
+    if args.unencrypted_suffix:
+        unencrypted_suffix = args.unencrypted_suffix
+    elif 'sops' in tree and 'unencrypted_suffix' in tree['sops']:
+        unencrypted_suffix = tree['sops']['unencrypted_suffix']
+    else:
+        unencrypted_suffix = DEFAULT_UNENCRYPTED_SUFFIX
+
     if args.encrypt:
         # Encrypt mode: encrypt, display and exit
         key, tree = get_key(tree, need_key)
-        tree = walk_and_encrypt(tree, key)
+        tree = walk_and_encrypt(tree, key, unencrypted_suffix=unencrypted_suffix)
```

Comment:
Since we can only have one `UNENCRYPTED_SUFFIX` per invocation of sops, I think it would make the code more readable to just rely on a global variable, instead of passing the value around.